templates/super/rules/controllers/log/view.php

```diff
@@ -0,0 +1,293 @@
+                                    </div>
+                                </div>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+            <tr>
+                <td colspan="2">
+                    <p class="text-left"><i class='ml-2 fa fa-comment' title=" <?php echo xla("Select feedback icon to show feedback") ?>"></i> <?php echo xlt("Rule has user provided feedback, select icon to show feedback"); ?></p>
+                </td>
+            </tr>
+        </table>
+
+    </div>  <!-- end of search parameters -->
+
+    <br />
+
+    <?php if (!empty($search) && ($search == 1)) { ?>
+        <div id="report_results">
+            <table class="table">
+
+                <thead>
+                <th align='center'>
+                    <?php echo xlt('Date'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('Patient ID'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('User ID'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('Facility ID'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('Category'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('All Alerts'); ?>
+                </th>
+
+                <th align='center'>
+                    <?php echo xlt('New Alerts'); ?>
+                </th>
+
+                </thead>
+                <tbody>  <!-- added for better print-ability -->
+                <?php foreach ($records as $row) : ?>
+                    <tr>
+                        <td><?php echo text(oeFormatDateTime($row['date'], "global", true)); ?></td>
+                        <td><?php echo text($row['pid']); ?></td>
+                        <td><?php echo text($row['uid']); ?></td>
+                        <td><?php echo text($row['facility_id']); ?></td>
+                        <td><?php echo text($row['category_title']); ?></td>
+                        <td>
+                            <?php
+                            //list off all targets with rule information shown when hover
+                            foreach ($row['formatted_all_alerts'] as $alert) {
+                                if (!empty($alert['rule_action_category'])) {
+                                    echo "<span title='" . attr($alert['title']) . "'>" .
+                                        $alert['rule_action_category'] . ": " . $alert['rule_action'] .
+                                        " (" . $alert['due_status'] . ")" .
+                                        "</span>";
+                                    // should never be empty... but just in case
+                                } else if (!empty($alert['text'])) {
+                                    echo $alert['text'];
```

Comment:
just making sure above alert element does not need to be escaped

Reply:
Fixed